akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/RoutersSpec.scala

```diff
@@ -27,7 +27,7 @@ class RoutersSpec extends ScalaTestWithActorTestKit("""
   def compileOnlyApiCoverage(): Unit = {
     Routers.group(ServiceKey[String]("key")).withRandomRouting().withRoundRobinRouting()
 
-    Routers.pool(10)(Behavior.empty[Any]).withRandomRouting().withRoundRobinRouting()
+    Routers.pool(10)(() => Behavior.empty[Any]).withRandomRouting().withRoundRobinRouting()
```

Comment:
maybe write that on two separate lines, since that is not really intended usage:
```
Routers.pool(10)(() => Behavior.empty[Any]).withRandomRouting()
Routers.pool(10)(() => Behavior.empty[Any]).withRoundRobinRouting()
```